CHANGELOG.md

```diff
@@ -28,6 +28,7 @@ stdout/stderr and `tt` logs go to `tt.log` file.
 - Ignore app-instance delimiters for Tarantool 3.0 instances.
 - Don't use dash as an app-instance delimiter. At the same time, `cartridge_app-stateboard`
 treated as a special case.
+- ``tt cat``: ``Result of cat`` now prints into stderr not stdout.
```

Comment:
Can we not be so specific about the text? `tt cat`: all diagnostic messages are printed to stderr.